### PR TITLE
#729 Fixed errors on Qualifications page

### PR DESCRIPTION
**Pull request by mbrookeswebdev**
Please check that errors are gone - 
qualifications

### CODE REVIEW

**Review by warrensearle — APPROVED** [review]
👍

**Comment by HalcyonJAC** [comment]
I checked out the `develop` branch and reproduced the bug, then checked out this branch and the bug has gone away.

Looks good to me :)

**Review by HalcyonJAC — APPROVED** [review]
I checked out the develop branch and reproduced the bug, then checked out this branch and the bug has gone away.

Looks good to me :)

**Comment by mbrookeswebdev** [comment]
@HalcyonJAC Well spotted :) I've made the changes.

To test: please fill in details on Qualification page, if you select "Barrister" and check "I did not complete pupillage", a text box should display asking for more details. Input details into the box and save. Please check that this info is displayed on the application Review page and on Admin in the Applications->Applied->(Any candidate)->Full info tab.

Qualifications2
Qualifications

**Review by HalcyonJAC — APPROVED** [review]
looks good to me :)

File: src/components/RepeatableFields/Qualification.vue [hunk 1]
```diff
@@ -109,6 +109,9 @@ export default {
     qualificationDetails() {
       return `qualification_details_${this.index}`;
     },
+    qualificationNotComplete() {
+      return `qualification_details_${this.index}`;
```
[the remark this answers is not in the document]
Comment (mbrookeswebdev) [reply src/components/RepeatableFields/Qualification.vue]: Done.

**Review by tomlovesgithub — APPROVED** [review]
Just want to be sure this matches the data model prescribed to the ```details```  in admin

**Review by warrensearle — APPROVED** [review]
Looks even better 👍

**Pull request merged**